docs/02.API-REFERENCE.md

```diff
@@ -212,6 +212,7 @@ Option bits for [jerry_parse](#jerry_parse) and
 
  - JERRY_PARSE_NO_OPTS - no options passed
  - JERRY_PARSE_STRICT_MODE - enable strict mode
+ - JERRY_PARSE_MODULE - parse source as an ECMAScript module
```

Comment:
A note is required for the new option below, like :`*Changed in version[[NEXT_RELEASE]]* : ...`